ruby/vampires.rb

```diff
@@ -1,4 +1,4 @@
-p "How many employees will be processed?"
+  p "How many employees will be processed?"
   number_employees = gets.chomp.to_i
 
 loop_counter = 0

```

Comment:
On line 12 you can refactor your name check to a single `if`

`if name == "Drake Cula" || name == "Tu Fang"`